HPCC-15045 DOCS:Document EclWatch WUPublish update super files option

g-pan opened the pull request:
FIX HPCC-15045 DOCS:Document EclWatch WUPublish update super files option
add to doc and update images.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com

@JamesDeFabia please review
@afishbeck please review

g-pan committed:
HPCC-15045 DOCS:Document EclWatch WUPublish update super files option

FIX HPCC-15045 DOCS:Document EclWatch WUPublish update super files option
add to doc and update images.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml:
```diff
@@ -480,6 +480,16 @@
+            <listitem>
+              <para><emphasis role="bold">Update Super Files:</emphasis>
+              Affects the publish action behavior when a query using
+              superfiles is published with a remote dali ip or the superfile
```
Dali should be proper case and IP should be cap (Dali IP)

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml:
```diff
@@ -480,6 +480,16 @@
+              superfiles is published with a remote dali ip or the superfile
+              is foreign. If it is not checked a referenced superfile that
+              doesn't already exist locally is created and copied from the
+              remote dali. If it is checked then the superfile is overwritten
```
Dali should be proper case

afishbeck replied:
Hmmmm.  I'm not sure about the wording because is sounds a bit either or.  That is:

1. If it is not checked a referenced superfile that doesn't already exist locally is created and copied from the remote dali. 
2. If it is checked then the superfile is overwritten

#1 always happens no matter what.
#2 only happens if checked.

So something like:

> Affects the publish action behavior.  When a query using superfiles is published with a remote dali ip or the superfile is foreign, super files will be copied from the remote dali.

> If the superfiles already exist locally then if this field is checked the current definition will be overwritten.  If it is not checked the current definition will not be changed.

But please improve the wording I didn't spend much time thinking through it.

JamesDeFabia commented:
Otherwise, looks good. But please allow @afishbeck to have the final say

g-pan committed:
HPCC-15045 DOCS:Document EclWatch WUPublish update super files option

FIX HPCC-15045 DOCS:Document EclWatch WUPublish update super files option
add to doc and update images.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-15045 DOCS:Document EclWatch WUPublish update super files option

FIX HPCC-15045 DOCS:Document EclWatch WUPublish update super files option
add to doc and update images.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK
